### PR TITLE
US03-03: googlemap page Haras

### PR DESCRIPTION
**Pull request by ChristianOlmedo**

### CODE REVIEW

**Review by NEL45 — DISMISSED** [review]
Composant fonctioned

File: public/assets/css/haras_googlemap.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.haras-googlemap {
+    display: flex;
+    justify-content: center;
+    border: 1px solid var(--primary-color);
+    border-radius: 0.5em;
+    padding: 0.1em;
```
Comment (LucyMerlier) [inline public/assets/css/haras_googlemap.css]: magic values

File: public/assets/css/haras_googlemap.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+}
+
+@media screen and (max-width: 800px) {
+    .haras-googlemap {
+        margin: 3em;
+    }
+}
```
Comment (LucyMerlier) [inline public/assets/css/haras_googlemap.css]: magic value + media query may be avoidable using bootstrap i think

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: public/assets/css/haras_googlemap.css (hunk 1 above)
Comment (ChristianOlmedo) [reply public/assets/css/haras_googlemap.css]: done

File: public/assets/css/haras_googlemap.css (hunk 2 above)
Comment (ChristianOlmedo) [reply public/assets/css/haras_googlemap.css]: done

**Review by bouboumael — APPROVED** [review]
ok for me

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**